backends/p4tools/modules/smith/util/util.cpp

```diff
@@ -58,4 +59,19 @@ std::string getRandomString(size_t len) {
     return ret;
 }
 
+/// Generate an arbitrary variable name for loop control.
+std::string generateLoopControlVariable() {
```

Comment:
We already have a function to generate a new string, why use this one?

Reply:
> We already have a function to generate a new string, why use this one?

I can't recall exactly what the reason was. It could be the case that I wrongly utilized the function so that it triggered some errors, or I was mind-boggling and sticking with the utility functions I wrote. But anyway, I changed it to the expression using `P4Tools::P4Smith::getRandomString(1)`.